File: python-pr/52/python/code-quality-naming/position_manager_44.py
# {fact rule=code-quality-naming@v1.0 defects=0}
x_coordinate, y_coordinate, z_coordinate = 0, 0, 0

def move_to_position(delta_x, delta_y):
    global x_coordinate, y_coordinate
    x_coordinate += delta_x
    y_coordinate += delta_y
# {/fact}

File: java-pr/49/java/naming/PositionManager44.java
// {fact rule=naming@v1.0 defects=0}
import java.util.*;

public class PositionManager44 {
    private int xCoordinate, yCoordinate, zCoordinate;
    
    public void moveToPosition(int deltaX, int deltaY) {
        xCoordinate += deltaX;
        yCoordinate += deltaY;
    }
}
// {/fact}